crypto/modes/gcm128.c

```diff
@@ -689,6 +697,13 @@ void gcm_init_p8(u128 Htable[16], const u64 Xi[2]);
 void gcm_gmult_p8(u64 Xi[2], const u128 Htable[16]);
 void gcm_ghash_p8(u64 Xi[2], const u128 Htable[16], const u8 *inp,
                   size_t len);
+# elif defined(OPENSSL_CPUID_OBJ) && defined(__riscv)
+#  include "crypto/riscv_arch.h"
+#  define GHASH_ASM_RISCV
+#  define GCM_FUNCREF_4BIT
+#  undef  GHASH
+void gcm_init_clmul_rv64gc_zbb_zbc(u128 Htable[16], const u64 Xi[2]);
```

Comment:
About the function name, I wonder whether the asm snippet needs the whole `rv64gc` to work, I think `rv64i` is sufficient?

Reply:
I agree. I will change the name to reference `rv64i`.